matching-service/clientScript.js

```diff
@@ -0,0 +1,12 @@
+import { io } from "socket.io-client";
```

Comment:
Hey @brelkh , not sure if this clientScript.js is needed anymore, since we're integrating frontend with backend